jenkins_node_scanner.py

```diff
@@ -158,6 +158,27 @@ def get_node_ip(master, node):
     return description.split('from ')[1].split(' :')[0]
 
 
+def get_node_info(master, target_ports, exclude_regex):
```

Comment:
This function return several items in a list, so it should be named `get_node_infos` or `get_nodes_info`.

Reply:
Agreed. I prefer the first option. Fixed in 6ebf4502ce8450894f658b07efd5d04cbbe8d1fa.